static/app/constants/index.tsx

```diff
@@ -318,6 +326,14 @@ export const DATA_CATEGORY_INFO = {
     titleName: t('Cron Monitors'),
     uid: 13,
   },
+  [DataCategoryExact.PROFILE_DURATION]: {
+    name: DataCategoryExact.PROFILE_DURATION,
+    apiName: 'profileSeconds',
+    plural: 'profileSeconds',
+    displayName: 'profile seconds',
+    titleName: t('Profile Seconds'),
+    uid: 17,
+  },
   [DataCategoryExact.METRIC_SECOND]: {
     name: DataCategoryExact.METRIC_SECOND,
     apiName: 'metricSecond',
```

Comment:
@scttcper there were some adjustments to the metrics category - would you be able to add those updates to this PR? 

```
[DataCategoryExact.METRIC_SECOND]: {
    name: DataCategoryExact.METRIC_SECOND,
    apiName: 'metricSeconds',
    plural: 'metric hours',
    displayName: 'metric hours',
    titleName: t('Metrics Hours'),
    uid: 19,
  },
  ```